Dockerfile

```diff
@@ -55,10 +55,12 @@ RUN set -ex; \
 RUN set -ex; \
     mkdir -p /var/lib/pulp/artifact \
              /var/lib/pulp/tmp \
+             /.ansible/tmp \
     && chown -R ${USER_NAME}:${USER_GROUP} \
         /app \
         /venv \
         /var/lib/pulp \
+        /.ansible/tmp \
```

Comment:
Can you configure it to use different directory?
I think it should be declared as a VOLUME.

Reply:
in galaxy-dev we added `ansible.cfg` to make it use `/tmp/ansible`, we can do the same and make that a VOLUME

Reply:
Please do it.

Reply:
I made the updates - I only added the volume to the pulp worker - let me know how these changes look